### ISSUE TITLE
Batch Norm - prevent subtraction of mean

### ISSUE THREAD

**Issue by simongraham** [tensorpack/tensorpack#1404]
Thanks for the amazing work with Tensorpack.

With the BatchNorm function - I want to use it such that we do not subtract the mean, but only divide by the standard deviation. i.e x' = x / stddev instead of x' = (x-mu) / stddev. Is there a simple way to do this in the current code? I know that there are options for whether to use scale and shift, but it looks a little more complicated if we don't want to subtract the mean.

I really appreciate your time looking at this.

**Comment by ppwwyyxx** [comment]
You can implement your customized version of the layer and use it.
See https://tensorpack.readthedocs.io/tutorial/extend/model.html
